### DISCUSSION TITLE
Running the axelor from the source code

### DISCUSSION THREAD

**Discussion by sunflashke** [community/community#110938] [Programming Help]
### Select Topic Area

Question

### Body

 I would like to get help in running the axelor from the source code. I need help in getting the code running ,.. i get this error below
 
* What went wrong:
Plugin [id: 'com.axelor.app', version: '8.0.3'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'com.axelor.app:com.axelor.app.gradle.plugin:8.0.3')
  Searched in the following repositories:
    MavenRepo

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sunflashke ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ethanappgenius** [comment]
It appears that you're encountering an error while trying to run Axelor from the source code due to a missing plugin. The error message indicates that the plugin "com.axelor.app" version 8.0.3 cannot be found in the specified repositories. To resolve this issue, ensure that your project's build configuration references the correct plugin version and that the plugin repository is accessible. You may need to update your Gradle configuration or check for any typos in the plugin declaration. Additionally, verify that your internet connection is stable and that the Maven repository hosting the plugin is accessible. By addressing these factors, you should be able to resolve the error and successfully run Axelor from the source code.

**Comment by sunflashke** [comment]
Can anyone guide me on how to fix this? Or install the plugin?
